.github/workflows/build-macos.yml

```diff
@@ -12,51 +12,50 @@ env:
   CARGO_TERM_COLOR: always
   RUST_BACKTRACE: short
   TERM: xterm-256color
-  BINARYEN_VERSION: version_111
 
 jobs:
-  x86:
-    runs-on: macos-latest
+  macos:
     strategy:
-      matrix: ${{ fromJson(inputs.matrix) }}
+      matrix:
+        profiles: ${{ fromJson(inputs.profiles) }}
+        arch:
+          - name: aarch64
+            image: macos-14
+          - name: x86
+            image: macos-14-large
       fail-fast: false
-    name: macos-x86 (${{ matrix.profile }})
+    runs-on: ${{ matrix.arch.image }}
+    name: macos-${{ matrix.arch.name }} (${{ matrix.profiles.name }})
```

Comment:
rename `x86` https://github.com/gear-tech/gear/blob/master/.github/actions/label/build.js#L11 with `macos-x86` (or `macos-aarch64` ?), need to discuss which arch should be enabled on labelling `forcemacos`

Reply:
I and @breathx vote to run both of them